docker-image-run-py/runpy

```diff
@@ -29,7 +29,17 @@ import socketserver
 import json
 import sys
 import io
-from code_runpy_backend import Struct, run_code, package_exception
+try:
+    from code_runpy_backend import Struct, run_code, package_exception
+except ImportError:
+    # When running tests
+    import os
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir)))
+    os.environ["DJANGO_SETTINGS_MODULE"] = "relate.settings"
+    import django
+    django.setup()
+    from course.page.code_runpy_backend import Struct, run_code, package_exception
```

Comment:
Can you explain why this is here? (in a comment)?

Reply:
As in, why the `django.setup()` hackery? If we're running tests, I'd imagine Django is already set up?

Reply:
Removed.